crates/aptos/README.md

```diff
@@ -11,10 +11,196 @@ With Cargo:
+```bash
+aptos init 
+Aptos already initialized, do you want to overwrite the existing config? [yes/no] >
+yes
+Enter your private key as a hex literal (0x...) [No input: Generate new key]
+
+No key given, generating key...
+Aptos is now set up!  Run `aptos help` for more information about commands
+{
+  "Result": "Success"
+}
+```
+
+#### Step 3) Changing config later
+You can always change it by using `aptos init` again.  Or you can manually edit `.aptos/config.yaml`.
+Currently, this configuration is tied to the directory you're in.
+
+### Create a new account with faucet
+
+We can create a new account with faucet and have it funded accordingly.
+
+#### Step 1) Generate a Private key
+
+You can skip this step if you already have a private key
+
+```bash
+aptos key generate --output-file my-key.key
+{
+  "Result": {
+    "PrivateKey Path": "my-key.key",
+    "PublicKey Path": "my-key.key.pub"
+  }
+}
+```
+
+#### Step 2) Retrieve ChainId
+```bash
+curl https://devnet.aptoslabs.com
+{"chain_id":8,"epoch":1,"ledger_version":"1154713","ledger_timestamp":"1650564773099386"}
+```
+
+#### Step 3) Create with faucet
+```bash
+aptos account create --use-faucet --chain-id 8 --public-key-file my-key.key.pub
+Account Created at 4978384F9C4B68AC5F907913CD5B36F9FF915E3D0F81C55C4A6CDFE4E7386BCF
+```
+
+### Listing resources in an account
+
+You can list resources in an account straight from the command line.
+
+```bash
+aptos account list --account 0000000000000000000000000000000000000000000000000000000000000001
+{
+  "Result": [
+    {
+      "authentication_key": "0x5f0f53b1394364832e78add00c7412be0845cd583caa36ffb1a2bffbf1cbaf83",
+      "self_address": "0x1",
+      "sequence_number": "0"
+    }
+  ]
+}
+```
+
+### Generating a Peer config
+
+For allowing others to connect to your node, you'll need to generate a peer_config.
+
+```bash
+aptos key extract-peer --private-key-file my-key.key --output-file peer_config.yaml
+{
+  "Result": {
+    "8946741e5c907c43c9e042b3739993f32904723f8e2d1491564d38959b59ac71": {
+      "addresses": [],
+      "keys": [
+        "0x8946741e5c907c43c9e042b3739993f32904723f8e2d1491564d38959b59ac71"
+      ],
+      "role": "Upstream"
+    }
+  }
+}
+```
+
+This creates a `peer_config.yaml` file like this:
+```bash
+---
+8946741e5c907c43c9e042b3739993f32904723f8e2d1491564d38959b59ac71:
+  addresses: []
+  keys:
+    - "0x8946741e5c907c43c9e042b3739993f32904723f8e2d1491564d38959b59ac71"
+  role: Upstream
+```
+
+After this point, you'd want to fill in your address accordingly.
+
+### Compiling Move
+
+The `aptos` CLI can be used to compile a Move package locally.
+In this example, we'll use the `HelloBlockchain` in [move-examples](../../aptos-move/move-examples/).
+
+```bash
+aptos move compile --package-dir aptos-move/move-examples/
+{
+  "Result": [
+    "000000000000000000000000000000000000000000000000000000000000E110::Message"
+  ]
+}
+```
+
+### Compiling & Unit Testing Move
```

Comment:
It would be very cool to show that we can actually debug and print stack trace when testing the move code. we can use ‘testing’ branch in toml when testing the move files ([natives](https://github.com/move-language/move/blob/main/language/move-stdlib/src/natives/mod.rs#L39)).

Reply:
That's a great idea!  But I think better outside of this as probably a tutorial.